src/adapters/poi/idunn_poi.js

```diff
@@ -87,6 +87,36 @@ export default class IdunnPoi extends Poi {
     }
   }
 
+  static async poiEventLoad(bbox, size, category, query) {
+    const url = `${serviceConfig.idunn.url}/v1/events`;
+    const requestParams = { bbox, size };
+    if (category) {
+      requestParams['category'] = category;
+    }
+    if (query) {
```

Comment:
So is this parameter useful after all?

Reply:
no, I cleaned it up but forgot to ûsh my last commit due to a surprise meeting :)